readthedocs/rtd_tests/tests/test_doc_serving.py

```diff
@@ -303,6 +313,16 @@ def test_sitemap_xml(self):
         # in language and country value. (zh_CN should be zh-CN)
         self.assertContains(response, 'zh-CN')
 
+        # External Versions should not be in the sitemap_xml.
```

Comment:
We may want to add all the external versions into the `robots.txt` file with Deny: https://github.com/rtfd/readthedocs.org/blob/3d9992ce15e8a944c98d537b40ab779a9b8b40e3/readthedocs/core/views/serve.py#L336-L339

Reply:
I believe we should be able to just `Deny` the entire domain we serve them from, when we're done. That should be much simpler.

Reply:
@ericholscher agree. I'm not sure if we already decide where to serve them from, but apart from that, I think that robots should be considered independently that decision.